### PR TITLE
TM-3733 position search org-location-label

### PR DESCRIPTION
**Pull request by andrews-john**
added org label. added css classes to org label and org text to assist with styling

### CODE REVIEW

**Review by andrews-john — COMMENTED** [review]
Adding that the org data is coming in through props on ResultsCard.jsx

File: src/sass/_results.scss [hunk 1]
```diff
@@ -135,6 +135,13 @@ $results-container-width: 100% - $filter-container-width;
     margin-bottom: 1rem;
     margin-top: .2rem;
 
+    .location-label{
+      width: 20%;
+    }
+
+    .location-text{
+      width: 90%;
+    }
```
Comment (SidelineCory24) [inline src/sass/_results.scss:144]: can remove this, the styling is already setup to move the `Org` to a new line depending on screen size and/or character length of the location text

File: src/sass/_results.scss [hunk 2]
```diff
@@ -162,7 +169,7 @@ $results-container-width: 100% - $filter-container-width;
     }
 
     dd {
-      padding: 0 0 0 .5rem;
+      padding: 0 0 0 .5em;
```
Comment (SidelineCory24) [inline src/sass/_results.scss:172]: can change this back to `.5rem`

File: src/Components/ResultsCard/ResultsCard.jsx [hunk 3]
```diff
@@ -231,7 +231,7 @@ class ResultsCard extends Component {
         );
 
     const headingBottom = !isTandem ?
-      <><dt>Location:</dt><dd>{post}</dd></>
+      <><dt className="location-label">Location (Org):</dt><dd className="location-text">{post}</dd></>
```
Comment (SidelineCory24) [inline src/Components/ResultsCard/ResultsCard.jsx:234]: classNames can be removed now since this styling is not being used

**Comment by SidelineCory24** [comment]
Need to adjust styling for smaller screen size, starts to look like this around 1200px, look at how we use `@media screen and` in the .scss files to account for this screen size
image

File: src/sass/_results.scss [hunk 4]
```diff
@@ -135,6 +135,13 @@ $results-container-width: 100% - $filter-container-width;
     margin-bottom: 1rem;
     margin-top: .2rem;
 
+    .location-label{
+      width: 20%;
+    }
+
+    .location-text{
+      width: 90%;
```
Comment (SidelineCory24) [inline src/sass/_results.scss:143]: changing to 95% will keep the spacing the same as before

File: src/sass/_results.scss [hunk 5]
```diff
@@ -162,7 +169,7 @@ $results-container-width: 100% - $filter-container-width;
     }
 
     dd {
-      padding: 0 0 0 .5rem;
```
Comment (SidelineCory24) [inline src/sass/_results.scss:165]: can change back to `.5rem` after doing [this](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2417/files#r1056614379)

**Review by SidelineCory24 — APPROVED** [review]
Looks good to go!🚀

**Pull request merged**